docs/user/editions/plasma/configuration.md

```diff
@@ -108,7 +114,9 @@ export XMODIFIERS=@im=ibus
    1. Open **IBus Preferences** from the application launcher.
    2. In the **Input Method** tab, add the keyboard layout you need.
    :::caution Important
+
    Some keyboard layouts require [installing additional packages](../../software/localization/ibus.md).
+
    :::
```

Comment:
This looks like a stray :::

Other than that, this PR is good to go. Great work.
Once this one thing is resolved, I can approve and merge this today.

Reply:
Corrected formatting.